articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md

```diff
@@ -0,0 +1,72 @@
+The engineering field has evolved rapidly over the past couple of years. Physical models are developed to show the final product design model. Today, engineering relies on software and machine tools. In the design process, engineers use the following techniques;
+
+- Computer-assisted design – It is technology in design and technical documentation. It involves the creation of computer models guided by geometrical parameters.
+- Computational fluid dynamics – It is used in design to check agreement with the standards and structural integrity.
+- Finite-element analysis – It is a tool for the product design process to make development analysis and ensure prompt implementation of analysis results.
+
+However, despite reliance on those techniques, engineers are still in charge of the design process. There is a new trend of artificial intelligence and innovations emergence, leading to system automation. The article will show the engineers' importance in the design process and the engineer's future.
+
+### System design
+
+It is the process of defining architecture, modules, and components and aligning their interface with the data system to the project's specifications. A systemic approach is essential for a coherent and well-running system. In system design, a designer uses modeling languages for information and knowledge expression. Design can be either graphical or textual modeling languages.
+
+#### Elements of a system
+
+The below are primary elements of a system;
+
+- Architecture – It is the backbone of a system, defines the structure, behavior, and view of a course.
+- Modules – Smallest units in a system, each is handling a specific task. Several modules together make a system.
+- Components – They are made up of modules; they provide a particular group function.
+- Interfaces – Boundaries upon which components share information and relate.
+- Data – A collection of facts and information.
+
+#### Significant tasks performed during the design process.
+
+- Initialize design definition – It is the planning stage; the designer identifies and determines the technologies to be used on the project.
+- Design Characteristics – It is the stage for defining each system element's architecture and characteristics; also, the definition of the viability.
+- Alternatives for obtaining system elements
+- Managing the design
+
+### [Artificial Intelligence (AI)](https://www.section.io/engineering-education/artificial-intelligence-future/)
+
+It is human intelligence in machines that are programmed to think like a machine like a human and their actions. A computer uses machine learning to determine how they respond to these situations.
```

Comment:
This definition is erroneous. AI is not human intelligence. Be very careful with the definitions you give.